backend/ibutsu_server/db/models.py

```diff
@@ -164,7 +177,66 @@ class WidgetConfig(Model, ModelMixin):
     navigable = Column(Boolean, index=True)
     params = Column(mutable_json_type(dbtype=PortableJSON()))
     project_id = Column(PortableUUID(), ForeignKey("projects.id"), index=True)
+    dashboard_id = Column(PortableUUID(), ForeignKey("dashboards.id"), index=True)
     title = Column(Text, index=True)
     type = Column(Text, index=True)
     weight = Column(Integer, index=True)
     widget = Column(Text, index=True)
+
+
+class User(Model, ModelMixin):
+    __tablename__ = "users"
+    email = Column(Text, index=True, nullable=False, unique=True)
+    password = Column(Text, nullable=False)
+    name = Column(Text)
+    project_id = Column(PortableUUID(), ForeignKey("projects.id"), index=True)
```

Comment:
I expect a user could be part of many projects. I'm thinking many-to-many here? Or getting rid of the project_id and just having `group_id`?

Reply:
Good point, I'll set that up.

Reply:
Converted this to a M-N relationship.